2020-09-07 15:21:54,469   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 15:41:11,787   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 15:44:53,655   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 15:49:02,519   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 15:51:05,306   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 16:00:35,004   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 16:11:50,365   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 16:13:05,296   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 16:17:50,618   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 16:21:04,862   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 18:59:14,109   ibmiotf.device.Client      CRITICAL Failed to connect to IBM Watson IoT Platform: crao1m.messaging.internetofthings.ibmcloud.com - [Errno 11001] getaddrinfo failed
2020-09-07 19:00:13,716   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:10:55,433   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:14:19,301   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:14:25,140   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:14:59,931   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:18:36,504   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:20:01,602   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:43:36,687   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:44:55,092   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:48:50,274   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:50:21,526   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:55:28,698   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 19:58:22,222   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:02:23,451   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:04:01,105   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:05:27,124   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:07:15,317   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:07:37,967   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:09:35,152   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:10:29,859   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:11:19,387   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:13:56,858   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:14:19,869   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:15:46,907   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:16:27,911   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:19:10,128   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 20:28:53,260   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 21:37:06,654   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 21:38:41,728   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 23:24:13,393   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 23:28:36,352   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 23:30:18,803   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 23:38:42,390   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-07 23:46:32,919   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 00:02:14,313   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:28:23,296   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:40:10,137   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:46:42,055   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:49:32,891   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:51:39,359   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 10:56:59,311   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 11:13:47,716   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 20:46:56,412   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 20:58:01,048   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-08 21:28:43,405   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-09 21:25:29,801   ibmiotf.device.Client      CRITICAL Failed to connect to IBM Watson IoT Platform: crao1m.messaging.internetofthings.ibmcloud.com - [Errno 11001] getaddrinfo failed
2020-09-09 21:26:50,765   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 11:31:45,969   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 11:37:58,454   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 14:18:43,812   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 14:48:30,724   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 15:32:01,374   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 15:38:21,183   ibmiotf.device.Client      ERROR   Unexpected disconnect from the IBM Watson IoT Platform: 1
2020-09-10 20:22:42,806   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 20:24:49,633   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 20:26:52,999   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 20:28:56,311   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 20:29:37,293   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
2020-09-10 20:30:48,669   ibmiotf.device.Client      INFO    Connected successfully: d:crao1m:raspberrypi:123456
